### PR TITLE
Interfaces: Add Real-time clock

### PR DESCRIPTION
**Pull request by albsod**

### CODE REVIEW

File: docs/interfaces/rtc.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: Real Time Clock (internal)
+tags:
+  - HMX
+  - C61
+  - MX-4
+  - T30
+  - T30 FR
+---
+
+The main CPU internal real time clock is `/dev/rtc0` and the time can be read out with the `hwclock` command.
```
Comment (RikardO-HM) [inline docs/interfaces/rtc.md:11]: a better way to read system time is timedatectl

File: docs/interfaces/rtc.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: Real Time Clock (internal)
```
Comment (RikardO-HM) [inline docs/interfaces/rtc.md:2]: move this to mx4  rtc under linux 3.1 
and create an new file rtc with timedatectl(https://gitlab.com/hostmobility/mx5-tools/-/blob/master/user_documentation/realtime_clock.md?ref_type=heads).

add chapter how to wakeup on the rtc. set alarm (for mx4 most user use go_to_sleep.sh)

**Comment by albsod** [comment]
@RikardO-HM, I made a number of changes. Please have a look again.

File: docs/interfaces/rtc.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+## Set system time to hardware clock
+
+
+```bash
+hwclock --systohc
+```
+
+## Set alarm (wake on RTC)
+
+- [Modern platforms](rtc_alarm.md)
```
Comment (RikardO-HM) [inline docs/interfaces/rtc.md:74]: link does no go anywhere
sugestion use 
sleeptime_seconds=30
RTC=/sys/class/rtc/rtc0/wakealarm; echo 0>$RTC && echo +${sleeptime_seconds} >$RTC && systemctl suspend
for HMX rtc1 will also work (mxv,c61,t30 rtc 0)

Comment (albsod) [reply docs/interfaces/rtc.md:74]: Thanks, fixed

**Pull request merged**